### DISCUSSION TITLE
Limit the permission user could do.

### DISCUSSION THREAD

**Discussion by atline** [labgrid-project/labgrid#900] [Q&A]
I understand labgrid use `hostname/username` to distinguish user.

And `labgrid-client -p xxx unlock` will be rejected if the place not acquired by me, that's great. But still, we could add `-k` to force kick off the user **by anyone**.

If possible in labgrid there is already something we could configure, with which we could specify just some user could kick off the place with `-k`? This could make our farm more controllable.

Thanks!

**Comment by Bastian-Krause** [comment]
labgrid is intended to be used in a _trusted environment_. Why would someone kick another user without asking? Seems like you are trying overcome a social obstacle with a technical solution.

Maybe I'm not getting your point, can you elaborate a bit?
